packages/react/src/PageHeading/PageHeading.tsx

```diff
@@ -8,10 +8,7 @@ import { forwardRef } from 'react'
 import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
 
 export type PageHeadingProps = {
-  /**
-   * De kleur van de titel
-   * Gebruik deze property om de titel in tegenovergestelde kleur te tonen.
-   */
+  /** Property to show the heading in the opposite color. */
```

Comment:
We could use ‘opposite’ instead of ‘inverse’ later, but I think ‘complementary’ has the connotation of ‘colours that work well with each other’ in everyday speech.

I’ve changed the description for the `inverseColor` prop.